civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1080,6 +1080,46 @@ function removeInactiveRoleDetails($results = []) {
   return $results;
 }
 
+/**
+ * Changes queries conditions of certain date fields which contains
+ * dates in "Y-m-d" format transforming those strings into timestamps
+ *
+ * @param view $view
+ * @param views_plugin_query_default $query
+ */
+function civihr_employee_portal_views_date_fields_to_timestamp(&$view, &$query, $dateFieldsToTimestamp) {
```

Comment:
what are we using this for? Seems a very complicated bit of logic that you had to write here, and there is no mention of it in the PR description

Reply:
Sure. I will described it in more details what is does.